bin/tf-reserve

```diff
@@ -3,7 +3,8 @@
 # testflinger-reserve
 tf-reserve() {
     queue="${1:-anything}"
-    data="${2:-jammy}"
+    data="${2:-noble}"
+    lp_user="${3:-$(whoami)}"
```

Comment:
Adding `export LP_USERNAME=hamistao` to your `~/.profile` or `~/.bashrc` feels easier than always having to specify your LP username as the 3rd parameter.
```suggestion
    lp_user="${LP_USERNAME:-$(whoami)}"
```

Reply:
Good idea! Since some people won't be using it all the time so I think it is nice to keep the parameter, what do you think about `lp_user="${3:-${LP_USERNAME:-$(whoami)}}"`?

Reply:
I feel that's providing too much choice and needlessly going to make it harder to ever add more positional parameters eventually.

Reply:
that's fair. In that case I think it makes sense to remove the `whoami` option because we don't have any reason to assume those would be the same values, leaving it as `${3:-LP_USERNAME}` or just `LP_USERNAME` if you think the parameter is not needed.

Reply:
I'd prefer to leave `$3` out of this :)
For consistency, you might want to use `LP_USER` instead of the longer form.